roombasim/pittras/ai/land_demo_controller.py

```diff
@@ -0,0 +1,36 @@
+'''
+land_demo_controller.py
+'''
+
+from roombasim.ai import Controller
+from roombasim.ai.task import TaskState
+
+class LandDemoController(Controller):
+
+    def setup(self):
+        self._switched = False
+
+        # TODO(mbilker): is there a way to initialize the drone to an
+        # arbitrary height above PITTRAS_LAND_HEIGHT_TOLERANCE instead of using
+        # TakeoffTask to get the height up
```

Comment:
The following one-liner should do the trick:

```py
(lambda cfg:(setattr(cfg,'AGENT',(lambda agent:(lambda *p:(lambda ref:(lambda _,x: x)(setattr(ref,'z_pos',1.5),ref))(agent(*p))))(getattr(cfg,'AGENT')))))(__import__('roombasim.config').config)
```

In all seriousness, I think I'll add some functionality to specify where to spawn the drone via the CLI.

Reply:
For the sake of testing, would it make more sense to have a random initialization in the controller that can be over-ridden by the CLI?

To do this in the controller, how do I access `environment` if that is even possible in the current implementation of RoombaSIM?

Reply:
If you're looking for a hack for testing purposes, you can just set `environment.agent.z_pos` to whatever you want the altitude to be

Reply:
Should I do that with the `LandTaskState.init` state in `LandTask`?

Reply:
You could do that, yes.  We really don't want that to be a permanent solution, probably don't want to merge code using that, but for testing it'll work

Reply:
I really would like a way to do this in the controller's `setup` method.

Reply:
It's not something for the controller to set

Reply:
Nvm, forgot what the controller is

Reply:
True, I could just use the `XYZTranslationTask` to navigate to the spot for setup and test the `LandTask` from there.

Reply:
Oh, I think its appropriate for the controller to set the location for startup, especially since the controller is what is specified by the command line.

Reply:
Yeah I forgot what RoombaSIM calls a "controller."  I still don't think it's a great idea to let the controllers set the position directly, it's already a parameter for the agent on startup we would just have to add a parameter in the cli

Reply:
Added the option to specify on command line in the `add-location-option` branch, you can check it out and cherry-pick or rebase or merge it or whatever

Reply:
Nice, this looks like a good way to do it. @mbilker can you merge `add-location-option` into this branch?

Reply:
This comment seems a bit buried, I'll go ahead and merge this